packages/bierzo-wallet/src/routes/balance/index.e2e.spec.ts

```diff
@@ -55,4 +56,10 @@ withChainsDescribe('E2E > Balance route', (): void => {
     expect(secondBalance).toBe('10 CASH');
     expect(thirdBalance).toBe('10 ETH');
   }, 45000);
+
+  fit('should contain message to get username', async (): Promise<void> => {
```

Comment:
fit?

Reply:
Also, we do not have an e2e integration test for showing a real human readable address, right?

Reply:
Yes we don't, but how to show it in e2e test? I wanted to ask this too.

Reply:
In this case, let @webmaster128 know we need to have a ticket for having an e2e test where:

1. We signup in the app using the extension
2. We buy a human-readable address
3. We travel to the account balance view
4. We see the human-readable address just bought. 

Also, do the same and assure if we just reload, the page, we do login and the human-readable address is there by default.

Reply:
@albertandrejev @apanizo if you want to extract work into a separate ticket, please create that on your own. There is no need for me to think about every single tickets that is created, especially when you are the experts of the particular job.

Reply:
oh! Thanks for the clarification, thought everything had to go through you. Maybe we should involve Martin for validating them?

Reply:
I assume everything is valid by default. If I find something that makes no sense, we can discuss and invalidate/adjust/close on demand. You guys know what you are doing. I just take care of prioritization.

Reply:
https://github.com/iov-one/ponferrada/issues/405